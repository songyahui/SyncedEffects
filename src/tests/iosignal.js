"use hopscript"

const hh = require("hiphop");

const prg =
      <hh.module I=${{accessibility: hh.IN}} X Y>
	<hh.emit Y/>
      </hh.module>;

var m = new hh.ReactiveMachine(prg);
m.debug_emitted_func = console.log

m.inputAndReact("I")
m.inputAndReact("X", 15)
m.react()
m.inputAndReact("Y")
