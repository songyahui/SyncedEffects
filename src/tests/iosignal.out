[ 'Y' ]
[ 'X(15)' ]
[]
[ 'Y' ]
